ansible/roles/formplayer/templates/sentry.properties.j2

```diff
@@ -0,0 +1,4 @@
+{% if http_proxy_address is defined %}
+http.proxy.host={{ http_proxy_address }}
+http.proxy.port={{ http_proxy_port }}
```

Comment:
I assume these two variables exist?

Reply:
Danny, when you ASSUME you make an ASS of U and ME

Reply:
I found them here https://github.com/dimagi/commcarehq-ansible/blob/3567e2b0946f8c23fc9570e287868d52e2dbe261/ansible/vars/icds/icds_public.yml#L193-L193